frontend/talentsearch/src/js/components/GovernmentInfoForm/GovernmentInfoForm.tsx

```diff
@@ -263,39 +257,6 @@ export const GovernmentInfoForm: React.FunctionComponent<
           </div>
         </>
       )}
-      {govEmployee === "yes" &&
-        (govEmployeeStatus === GovEmployeeType.Term ||
-          govEmployeeStatus === GovEmployeeType.Indeterminate) && (
-          <p>
-            {intl.formatMessage({
-              defaultMessage:
-                "Please indicate if you are interested in lateral deployment or secondment. Learn more about this.",
-              description:
-                "Text blurb, asking about interest in deployment/secondment in the government info form",
```

Comment:
Should the corresponding strings in fr.json be removed as well for any `intl.formatMessage` removed?

Reply:
Good idea, removed with [remove secondment fr string](https://github.com/GCTC-NTGC/gc-digital-talent/pull/3698/commits/a2fbcbeb8b970b79a92c610c5a9821f732a2417e)